app/src/main/java/com/arygm/quickfix/ui/elements/QuickFixButton.kt

```diff
@@ -48,8 +55,21 @@ fun QuickFixButton(
         Row(
             modifier = Modifier.fillMaxWidth(),
             horizontalArrangement = horizontalArrangement,
+            verticalAlignment = Alignment.CenterVertically,
         ) {
+          leadingIcon?.let {
+            Image(
+                imageVector = it,
+                contentDescription = null,
+                modifier = Modifier.padding(end = 8.dp))
+          }
           Text(text = buttonText, style = textStyle, color = textColor)
+          trailingIcon?.let {
+            Image(
+                imageVector = it,
+                contentDescription = null,
+                modifier = Modifier.padding(start = 8.dp))
+          }
```

Comment:
Great change ! This will allow all the team to create all the the needed buttons.